handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -120,23 +128,35 @@ func newTokenInfo(t *jwt.Token, timeBase time.Time) (*TokenInfo, error) {
 	}, nil
 }
 
-func claimAsStrings(t *jwt.Token, claim string) ([]string, bool) {
+func NewTokenInfo(t *jwt.Token, timeBase time.Time) (*TokenInfo, error) {
```

Comment:
Then put it into an interface to specify your behaviour and you probably don't need the type casts later.